atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -339,7 +339,8 @@ def get_logs(self):
             # Deleted once closed
             logfile = NamedTemporaryFile(prefix=self.build_id,
                                          suffix=".log",
-                                         mode='w')
+                                         mode='w',
+                                         encoding='UTF-8')
```

Comment:
Only available in python3. In python 2, there's codec.open, but not sure about temp file.